qiskit-superstaq/qiskit_superstaq/superstaq_job.py

```diff
@@ -97,45 +96,88 @@ def result(self, timeout: Optional[float] = None, wait: float = 5) -> qiskit.res
                 "qobj_id": -1,
                 "backend_name": self._backend._configuration.backend_name,
                 "backend_version": self._backend._configuration.backend_version,
-                "success": True,
+                "success": self._overall_status == "Done",
+                "status": self._overall_status,
```

Comment:
think these ones should still be
```suggestion
                    "success": result["status"] == "Done",
                    "status": result["status"],
```
(that way individual sub-jobs can have their own statuses when the overall status is "Failed")